scripts/test/Muon/loading_tab/loadwidget_presenter_multiple_file_test.py

```diff
@@ -65,7 +67,7 @@ def tearDown(self):
 
     def assert_model_contains_correct_loaded_data(self):
         # use sorted due to threads finishing at different times
-        self.assertEqual(sorted(self.presenter._model.filenames), ['Co-added'])
+        # self.assertEqual(sorted(self.presenter._model.filenames), ['Co-added'])
```

Comment:
can we delete this?

Reply:
I've uncommented this as don't know why it was commented out